tests/micro/arduino/conftest.py

```diff
@@ -48,13 +35,23 @@
 ).resolve()
 
 
+def arduino_micro_devices() -> dict:
```

Comment:
Can we cache this value? `arduino_micro_devices()` will probably be called several times by our test file.

Reply:
done.